### PR TITLE
Minor clean up after recent Netty upgrades

### PR DESCRIPTION
**Pull request by danbev**

### CODE REVIEW

**Comment by danbev** [comment]
@matzew I had some spare cycles while provisioning, would you mind taking a look at these changes?

File: server-netty/src/main/java/org/jboss/aerogear/webpush/netty/WebPushNettyServer.java [hunk 1]
```diff
@@ -93,11 +93,9 @@ private static SslContext createSslContext(final WebPushServerConfig config) thr
     }
 
     private static Protocol protocol(final WebPushServerConfig config) {
-        switch (config.protocol()) {
-            case ALPN:
-                return Protocol.ALPN;
-            default:
-                throw new IllegalStateException("Protocol not supported [" + config.protocol() + "]");
+        if (config.protocol() == WebPushServerConfig.Protocol.ALPN) {
```
Comment (idelpivnitskiy) [inline server-netty/src/main/java/org/jboss/aerogear/webpush/netty/WebPushNettyServer.java]: @danbev what about completely remove protocol from config and use only ALPN?

**Comment by idelpivnitskiy** [comment]
@danbev LGTM!
I also done some manual integration tests, everything works fine!

File: server-netty/src/main/java/org/jboss/aerogear/webpush/netty/WebPushNettyServer.java (hunk 1 above)
Comment (danbev) [inline server-netty/src/main/java/org/jboss/aerogear/webpush/netty/WebPushNettyServer.java]: Good point, I'll remove it.

**Comment by danbev** [comment]
@idelpivnitskiy I've removed the protocol. Mind taking a quick look? thx

**Comment by idelpivnitskiy** [comment]
@danbev everything looks and works good!)

**Pull request merged**